### ISSUE TITLE
rc5 release makes lib-python files not world readable

### ISSUE THREAD

**Issue by lucaswerkmeister** [graalvm/graalpython#13]
Hi! I publish the [graalpython-bin](https://aur.archlinux.org/packages/graalpython-bin/) AUR package, and while updating it to rc5, I noticed that using graalpython didn’t completely work anymore:

```
$ graalpython 
Please note: This Python implementation is in the very early stages, and can run little more than basic benchmarks at this point.
org.graalvm.launcher.Launcher$AbortException
Caused by: java.lang.RuntimeException: error while retrieving prompt
        at java.lang.Throwable.<init>(Throwable.java:287)
        at java.lang.Exception.<init>(Exception.java:84)
        at java.lang.RuntimeException.<init>(RuntimeException.java:80)
        at com.oracle.graal.python.shell.GraalPythonMain.getPrompt(GraalPythonMain.java:605)
        at com.oracle.graal.python.shell.GraalPythonMain.readEvalPrint(GraalPythonMain.java:516)
        at com.oracle.graal.python.shell.GraalPythonMain.launch(GraalPythonMain.java:199)
        at org.graalvm.launcher.AbstractLanguageLauncher.launch(AbstractLanguageLauncher.java:107)
        at org.graalvm.launcher.AbstractLanguageLauncher.launch(AbstractLanguageLauncher.java:54)
        at com.oracle.graal.python.shell.GraalPythonMain.main(GraalPythonMain.java:56)
        at com.oracle.svm.core.JavaMainWrapper.run(JavaMainWrapper.java:177)
Caused by: org.graalvm.polyglot.PolyglotException: OSError: /usr/lib/jvm/java-8-graal/jre/languages/python/lib-python/3/_pyio.py
        at <python>.open(Unknown)
        at <python>.FileIO.__init__(_io:345)
        at <python>.FileLoader.get_data(_bootstrap_external.py:841)
        at <python>.SourceLoader.get_code(_bootstrap_external.py:789)
        at <python>._LoaderBasics.exec_module(_bootstrap_external.py:682)
        at org.graalvm.polyglot.Context.eval(Context.java:313)
        at com.oracle.graal.python.shell.GraalPythonMain.getPrompt(GraalPythonMain.java:600)
        at com.oracle.graal.python.shell.GraalPythonMain.readEvalPrint(GraalPythonMain.java:516)
        at com.oracle.graal.python.shell.GraalPythonMain.launch(GraalPythonMain.java:199)
        at org.graalvm.launcher.AbstractLanguageLauncher.launch(AbstractLanguageLauncher.java:107)
        at org.graalvm.launcher.AbstractLanguageLauncher.launch(AbstractLanguageLauncher.java:54)
        at com.oracle.graal.python.shell.GraalPythonMain.main(GraalPythonMain.java:56)
        at com.oracle.svm.core.JavaMainWrapper.run(JavaMainWrapper.java:177)
        at <python>._load_unlocked(_bootstrap.py:662)
        at <python>._find_and_load_unlocked(_bootstrap.py:957)
        at <python>._find_and_load(_bootstrap.py:968)
        at <python>._gcd_import(_bootstrap.py:985)
        at <python>.__import__(_bootstrap.py:1066)
        at <python>.<module '__builtins_patches__'>(__builtins_patches__:48)
```

```
$ Rscript --polyglot -e 'eval.polyglot("python", "1")'
Error in eval.polyglot("python", "1") :
  Error while parsing: ImportError: /usr/lib/jvm/java-8-graal/jre/languages/python/lib-python/3/importlib/_bootstrap_external.py
```

As part of the installation process, the package applies the `META-INF/permissions` file found in the jar, and it seems that in the [rc5 jar](https://github.com/graalvm/graalpython/releases/download/vm-1.0.0-rc5/python-installable-ce-1.0.0-rc5-linux-amd64.jar), that file marks everything in `lib-python` as not readable by anyone other than `root`:

```
…
jre/languages/python/include/object.h = rw-rw-r--
jre/languages/python/include/pyctype.h = rw-rw-r--
jre/languages/python/include/pyconfig.h = rw-rw-r--
jre/languages/python/lib-python/3/sre_constants.py = rw-------
jre/languages/python/lib-python/3/symbol.py = rw-------
jre/languages/python/lib-python/3/abc.py = rw-------
jre/languages/python/lib-python/3/_sitebuiltins.py = rw-------
…
```

root can still run `graalpython` or run polyglot Python from other languages, but other users are no longer allowed to read the files with graalpython requires. In rc4 this was not a problem: there, the `permissions` file listed `lib-python` as world-readable (mode `rw-rw-r--`).

**Comment by lukasstadler** [comment]
That is indeed a problem - thanks for reporting this.
@ansalond - is this something to be solved on the Python side, or on the infrastrure side?

**Comment by gilles-duboscq** [comment]
The build issue that lead to this has been fixed by graalvm/mx@935ad0f7f234407dc05b95ade6651e3e04cdb917.
